src/main/java/com/google/sps/servlets/FirebaseAppInit.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.auth.FirebaseToken;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.io.FileInputStream;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.io.IOUtils;
+import java.io.*;
+import java.util.List;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.firebase.auth.FirebaseAuthException;
+
+public class FirebaseAppInit {
+
+    private String jsonFilePath = "/firebaseServiceAccount.json";
+
+    public FirebaseApp initializeFirebaseApp() throws IOException {
+
+        InputStream serviceAccount = FirebaseAppInit.class.getResourceAsStream(jsonFilePath);
+
```

Comment:
yes. it will throw an IOException here and a NullPointerException in the FirebaseAppInit.java (since the auth.oauth2.GoogleCredentials needs the data from the jsonFile to verify that our FirebaseApp is valid)